web_m2x_options/static/src/components/form.esm.js

```diff
@@ -304,6 +320,22 @@ Many2OneAvatarField.props = {
     nodeOptions: {type: Object, optional: true},
 };
 
+// Add props to field widgets throwing error
+
+Many2OneAvatarUserField.props = {
+    ...Many2OneAvatarUserField.props,
+    searchMore: {type: Boolean, optional: true},
+    nodeOptions: {type: Object, optional: true},
+};
+
+SaleOrderLineProductField.props = {
```

Comment:
There are many other classes which extend from Many2OneField (ex. ProjectMany2OneField). All of them face with such issue. 
Bad idea to patch them in a base module (a dependency required).

We have to look for a better idea.